blockdevice/probe/probe.go

```diff
@@ -84,6 +84,8 @@ func WithSingleResult() SelectOption {
 			return false, fmt.Errorf("got more than one blockdevice with provided criteria")
 		}
 
+		count++
```

Comment:
it never worked without this condition (???)

Reply:
lol, oops. Thanks for fixing that :)